stdlib/Random/src/RNGs.jl

```diff
@@ -331,14 +331,22 @@ end
 typeof_rng(::_GLOBAL_RNG) = TaskLocalRNG
 
 """
-    default_rng() -> rng
+    Random.default_rng() -> rng
 
-Return the default global random number generator (RNG).
+Return the default global random number generator (RNG), which is used by `rand`-related functions when
+no explicit RNG is provided.
+
+When the `Random` module is loaded, the default RNG is _randomly_ seeded, via [`Random.seed!()`](@ref):
+this means that each time a new julia session is started, the first call to `rand()` produces a different
+result, unless `seed!(seed)` is called first.
+
+It is thread-safe: distinct threads can safely call `rand`-related functions on `default_rng()` concurrently,
+e.g. `rand(default_rng())`.
 
 !!! note
-    The default RNG is an implementation detail. Across different versions of
-    Julia, you should not expect the default RNG to be always the same, nor that it will
-    return the same stream of random numbers for a given seed.
+    The type of the default RNG is an implementation detail. Across different versions of
+    Julia, you should not expect the default RNG to have always the same type, nor that it will
+    produce the same stream of random numbers for a given seed.
```

Comment:
```suggestion
    The type of the default RNG is an implementation detail. Across different versions of
    Julia, you should not expect the default RNG to always have the same type, nor that it will
    produce the same stream of random numbers for a given seed.
```